### PR TITLE
Bugfix: fixing screen rendering on some edge case + minor code refactoring + clarified --help

### PR DESCRIPTION
**Pull request by Lucas-C**
Take 2 after https://github.com/tehmaze/diagram/pull/18

### CODE REVIEW

File: test/test_random.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+cd $(dirname "${BASH_SOURCE[0]}")/..
+
+./bin/random 100 | python diagram.py
+./bin/random 100 | python diagram.py -H
+./bin/random 100 | python diagram.py -V
+./bin/random 100 | python diagram.py -b
+./bin/random 100 | python diagram.py -bH
+./bin/random 100 | python diagram.py -bV
+
+echo -e '1\n2\n3\n2' | diagram -H > output
```
Comment (tehmaze) [inline test/test_random.sh]: This should call `python diagram.py`

File: test/test_random.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+# We check that the number of lines in the output is [input size] + 1
+[ $(wc -l output | cut -d' ' -f1) -eq 5 ] || exit 1
```
Comment (tehmaze) [inline test/test_random.sh]: Please consider using a more portable `| awk '{print $1}'`

**Comment by tehmaze** [comment]
Thanks for the new PR!

**Comment by Lucas-C** [comment]
I made the requested changes :)

**Comment by Lucas-C** [comment]
Is there anything missing to merge this PR ?

**Comment by Lucas-C** [comment]
Any update on this PR @tehmaze ? :)
